File: src/main/java/bohdan/markatov/org/dto/project/ProjectResponseDto.java
package bohdan.markatov.org.dto.project;

import bohdan.markatov.org.dto.user.UserResponseDto;
import java.util.Set;
import lombok.Data;

@Data
public class ProjectResponseDto {
    private Long id;
    private String name;
    private String description;
    private UserResponseDto manager;
    private Set<UserResponseDto> teamMembers;
}

File: frontend/src/app/services/models/project-response-dto.ts
/* tslint:disable */
/* eslint-disable */
import { UserResponseDto } from '../models/user-response-dto';
export interface ProjectResponseDto {
  description?: string;
  id?: number;
  manager?: UserResponseDto;
  name?: string;
  teamMembers: Array<UserResponseDto>;
}
